fdbclient/S3BlobStore.actor.cpp

```diff
@@ -545,6 +545,8 @@ ACTOR Future<Void> updateSecret_impl(Reference<S3BlobStoreEndpoint> b) {
 						creds.key = apiKey;
 					else
 						continue;
+				} else {
```

Comment:
Good catch. This logic when porting from release-6.3 got a little mangled and introduced this bug. To match 6.3 we should remove the else block and change the creds initialization as

S3BlobStoreEndpoint::Credentials creds = b->credentials.get();

Reply:
Done.